tests/test_utils.py

```diff
@@ -91,15 +91,16 @@ def mock_validate_index(mocker):
     return mocker.patch.object(validate, "validate_index")
 
 
-def test_md5(mocker):
-    audio_file = b"audio1234"
+# TODO: rewrite this!
```

Comment:
Can you explain how or why to rewrite this?

Reply:
the test fails now, and we're patching builtings.open, instead of smart_open

Reply:
Thanks. Can you add that to the comment?